packages/material-ui/src/Divider/Divider.js

```diff
@@ -88,14 +154,28 @@ const Divider = React.forwardRef(function Divider(props, ref) {
           [classes.flexItem]: flexItem,
           [classes.light]: light,
           [classes.vertical]: orientation === 'vertical',
+          [classes.text]: children,
+          [classes.textVertical]: children && orientation === 'vertical',
+          [classes.textAlignRight]: textAlign === 'right' && orientation !== 'vertical',
+          [classes.textAlignLeft]: textAlign === 'left' && orientation !== 'vertical',
         },
         themeVariantsClasses,
         className,
       )}
       role={role}
       ref={ref}
       {...other}
-    />
+    >
+      {children ? (
+        <span
+          className={clsx(classes.spanText, {
```

Comment:
```suggestion
          className={clsx(classes.children, {
```

Reply:
Is there an opportunity for a more descriptive label, like wrapper, container, else?

Reply:
`wrapper` sounds good in my opinion 👍

Reply:
@ShehryarShoukat96 renaming `classes.children` to `classes.wrapper` and `classes.childrenVertical` to `classes.wrapperVertical` would be the last required change